m4_sessions/1708-inning/group_five.md

```diff
@@ -1,9 +1,21 @@
 # Group Five
 
-### Session Title:
+### Session Title: The Robots are Coming: The future of AI
 
 ### Session Outcomes: 
+- What does it mean to me as a programmer
+- Want people to consider how AI benefits or threatens their lives as devs
+- What AI is currently available, near future, and distant future
+- Understanding of what AI is (definitions and relevant vocabulary)
 
 ### Session Outline:
+- 10 to 15 min Intro (Icebreaker)
+- 20 min content video
+- Discuss the video
```

Comment:
Once you pick your video, create 3-5 discussion questions to get students thinking.  You might want to give them these questions before they see the video so they are thinking about it.